app/logic/__init__.py

```diff
@@ -218,3 +218,26 @@ def generate_signed_url():
                             props=filedata[relative_path])
         res_payload['filedata'][relative_path] = response.build_file_information()
     return res_payload
+
+
+#### helpers
+
+def validate_for_template(descriptor):
+    '''
+    Cheks if descriptor fields have expected type for template Eg:
```

Comment:
Probably it's better to rewrite:
"This function validates field types in the descriptor, e.g.,  `license` property should be a list."